File: webtruyen/toc.js
function execute(url) {
    var doc = Http.get(url).html();

    var el = doc.select(".list-chapter").last().select("ul h4 a")
    const data = [];
    for (var i = 0; i < el.size(); i++) {
        var e = el.get(i);
        data.push({
            name: e.text(),
            url: e.attr("href"),
            host: "https://webtruyen.com"
        })
    }

    return Response.success(data);
}

File: blogtruyen/toc.lua
local url = ...

local doc = http:get(url):html()
if doc ~= nil then
    local list = {}
    local el = doc:select("#list-chapters a")
    for i = el:size() - 1, 0, -1 do
        local e = el:get(i)
        local chap = {
            ["name"] = e:text(),
            ["url"] = e:attr("href"),
            ["host"] = "https://blogtruyen.vn",
        }
        table.insert(list, chap)
    end
    return response:success(list)
end
return nil